pkg/systemd/overview-cards/uncleanShutdownStatus.jsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import React, { useState } from 'react';
+import { Button } from "@patternfly/react-core/dist/esm/components/Button/index.js";
+import { Flex, FlexItem } from "@patternfly/react-core/dist/esm/layouts/Flex/index.js";
+import { Icon } from "@patternfly/react-core/dist/esm/components/Icon/index.js";
+import { ExclamationTriangleIcon } from "@patternfly/react-icons";
+
+import cockpit from "cockpit";
+
+import './uncleanShutdownStatus.css';
+import { useInit } from 'hooks';
+
+const _ = cockpit.gettext;
+
+export const UncleanShutdownStatus = () => {
+    const [uncleanShutdownId, setUncleanShutdownId] = useState(null);
+    const [uncleanShutdownStatusVisible, setUncleanShutdownStatusVisible] = useState(false);
+
+    useInit(() => {
+        cockpit.spawn(
+            ["last", "--system", "--limit=2", "--time-format=iso", "shutdown", "reboot"],
+            { environ: ["LC_ALL=C"], err: "message", }
+        ).then((data) => {
+            const previous_boot = data.split("\n")[1];
+            if (previous_boot === undefined)
+                return;
+
+            // "crash" on wtmpdb, "still running" on util-linux
+            if (!previous_boot.includes("crash") && !previous_boot.includes("still running"))
+                return;
+
+            const lines = previous_boot.split(/ +/);
+            const started = new Date(lines[4]);
+            if (isNaN(started)) {
```

Comment:
Checking the command myself I see:
```bash
❯ last --system --limit=2 --time-format=iso shutdown reboot
reboot   system boot  6.17.5-ba07.fc4* 2025-11-13T12:39:15+01:00   still running
shutdown system down  6.17.5-ba07.fc4* 2025-11-13T02:09:52+01:00 - 2025-11-13T12:39:15+01:00  (10:29)
```
I don't think the RegEx is stable enough. Not only that, if I understand this correctly getting the endtime wouldn't work due to `- `. Which we don't need here but still showcases it being icky
```suggestion
            const re = /(?<status>\S+) +(?<action>system \S+) +(?<kernel>\S+) +(?<startDate>\S+) +(?:(?<endDate>still running)|- (?<endDate>\S+)) *(?:\((\S+)\))?/g
            const match = re.exec(previous_boot);
            const started = new Date(match.groups["startDate"]);
            if (isNaN(started)) {
```